.travis.yml

```diff
@@ -2,10 +2,17 @@ language: node_js
 node_js:
   - 8.15.0
 
+before_deploy:
+  - yarn
+  - yarn build
+
 deploy:
+  deploy:
```

Comment:
Why is `deploy:` being repeated in lines #9 and #10?

Reply:
probably merge mistake 🤦‍♀️